index.js

```diff
@@ -62,15 +74,27 @@ module.exports = function(options) {
       requireName = camelizePluginName ? camelize(requireName) : requireName;
     }
 
-    if(lazy) {
-      Object.defineProperty(finalObject, requireName, {
-        get: function() {
-          return requireFn(name);
-        }
-      });
+    return requireName;
+  }
+
+  var scopeTest = new RegExp('^@');
+  var scopeDecomposition = new RegExp('^@(.+)/(.+)');
+
+  multimatch(names, pattern).forEach(function(name) {
+    var hasScope = scopeTest.test(name);
+    var decomposition = scopeDecomposition.exec(name);
```

Comment:
This is not really needed unless we are in a hasScope case. I'd vote to move this down and inline the "requireName" value, but that's totally a style choice. 

I doubt there would be any measurable performance concern.